src/push-service.js

```diff
@@ -1,76 +1,27 @@
-/**
- * Handle push events
- */
-self.addEventListener('push', function(event) {
-  if (!(self.Notification && self.Notification.permission === 'granted')) {
-    return;
-  }
-
-  // extract notification data if possible
-  var data = event.data ? event.data.json() : {};
-  var title = 'title' in data ? data.title : 'Meditation+';
-
-  // set a default icon if none was declared
-  if (!('icon' in data)) {
-    data.icon = './assets/icon/android-chrome-192x192.png';
-  }
-
-  // show notification
-  event.waitUntil(
-    self.registration.showNotification(title, data)
-  );
+// Give the service worker access to Firebase Messaging.
+// Note that you can only use Firebase Messaging here, other Firebase libraries
+// are not available in the service worker.
+importScripts('https://www.gstatic.com/firebasejs/4.5.0/firebase-app.js');
+importScripts('https://www.gstatic.com/firebasejs/4.5.0/firebase-messaging.js');
+
+// Initialize the Firebase app in the service worker by passing in the
+// messagingSenderId.
+firebase.initializeApp({
+  'messagingSenderId': '223027225088'
```

Comment:
What's this id?

Reply:
This branch is not really ready yet at all :D Will need more time for developing and testing.